apps/docs/app/ui/layout/wrapper/wrapper.css

```diff
@@ -1,20 +1,17 @@
 .hd-layout__wrapper {
-    width: min(100% - 2rem, var(--layout-width));
+    width: min(100% - 1rem, var(--layout-width));
```

Comment:
Keep `2rem` padding otherwise there's not enough space on the sides when you're on the mobile

Reply:
With 2 the layout break too soon, I kinda tried to stay on the 48 rem breakpoint, otherwise we'd have to go 48.75rem breakpoint which is odd.

Reply:
I think defining `box-sizing: border-box` solves the problem.

Reply:
Sadly not at one specific pixel breakpoints (768px) we get this issue: 
![image](https://github.com/user-attachments/assets/2594f797-34c5-4263-8a89-3021790befad)